perf/src/index.tsx

```diff
@@ -61,9 +62,15 @@ const renderCycle = async (
   return profilerMeasure
 }
 
-const satisfiesFilter = (componentName: string, filter: string) => {
-  return componentName.toLowerCase().startsWith(filter.toLowerCase())
-}
+const satisfiesFilter = (componentName: string, filter: string) =>
+  minimatch(componentName, filter || '*', {
+    matchBase: true,
+  })
+
+const sortObjectKeys = o =>
+  Object.keys(o)
+    .sort()
+    .reduce((r, k) => ((r[k] = o[k]), r), {})
```

Comment:
o_O

Reply:
Removed it at all, it confused two people...